<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite name="Failed suite [Default suite]" guice-stage="DEVELOPMENT">
  <test thread-count="5" name="Default test(failed)">
    <classes>
      <class name="zapcg.Capillary.LoginTestCases.OneTimeExecute_Login_LockOutTestCase">
        <methods>
          <include name="verify_ValidationMessage_For5Invalid_LoginAttempt"/>
          <include name="afterSuite"/>
          <include name="tearDown"/>
          <include name="initialize"/>
        </methods>
      </class> <!-- zapcg.Capillary.LoginTestCases.OneTimeExecute_Login_LockOutTestCase -->
    </classes>
  </test> <!-- Default test(failed) -->
</suite> <!-- Failed suite [Default suite] -->
